src/compas_ghpython/artists/circleartist.py

```diff
@@ -31,12 +31,33 @@ def draw(self):
         :class:`Rhino.Geometry.Circle`
 
         """
-        point = list(self.primitive.plane.point)
-        normal = list(self.primitive.plane.normal)
-        radius = self.primitive.radius
-        circles = [{'plane': [point, normal], 'radius': radius, 'color': self.color, 'name': self.name}]
+        circles = [self._get_args(self.primitive)]
         return compas_ghpython.draw_circles(circles)[0]
 
+    @staticmethod
+    def draw_collection(collection):
+        """Draw the collection of circles.
+
+        Parameters
+        ----------
+        collection : list of compas.geometry.Circle
+            A collection of ``Circle`` objects.
+
+        Returns
+        -------
+        list of :class:`Rhino.Geometry.Circle`
+
+        """
+        circles = [CircleArtist._get_args(primitive) for primitive in collection]
+        return compas_ghpython.draw_circles(circles)
+
+    @classmethod
+    def _get_args(cls, primitive):
```

Comment:
perhaps this should be `@staticmethod`. no need to pass `cls` as implicit first argument...